template.en.ja.json

```diff
@@ -67,6 +67,12 @@
             "projects-field": "",
             "text": "*Story*\nAs a <TI>type of user/persona</TI>, I want <TI>to perform some task</TI>, so that I can <TI>achieve some goal/benefit/value</TI>.\n\n*Details*\n<TI>Enter any details, clarifications, answers to questions, or points about implementation here.</TI>\n\n*Additional Information*\n<TI>Enter any background or references such as Stack Overflow, MSDN, blogs, etc. that may help with developing the feature.</TI>\n\n*Acceptance Criteria*\n<TI>Enter the conditions of satisfaction here. That is, the conditions that will satisfy the user/persona that the goal/benefit/value has been achieved.</TI>"
         },
+        {
+            "issuetype-field": "タスク",
+            "name": "TASK TEMPLATE",
+            "projects-field": "Design (DES)",
+            "text": "h3. 解決すべき課題\n- <TI>やることではなく、どういう問題を解決したいかを書いてください。</TI>\n\nh3. 経緯\n- <TI>どういう経緯で話が出てきたかを書いてください。経緯を聞いてみたらそもそも課題が違った、ということを防ぎます。</TI>\n\nh3. やりたいこと\n- <TI>解決策として何をやりたいかを書いてください。</TI>\n\nh3. 締切\n- <TI>わからない場合でも仮で書いてください。</TI>\n\nh3. イメージ\n- <TI>手書きや他のアプリのキャプチャがあれば貼ってください。会議で書いたホワイトボードなどの写真もあれば貼ってください。</TI>\n\nh3. 参考リスト/関係する人\n- <TI>関連するURL/Issue/人物などがあれば書いておいてください。</TI>\n"
```

Comment:
Preferred to be in English even though there are only Japanese colleagues.

Reply:
I agree to this opinion. By the way, setting English and Japanese both is good for me

Reply:
The original text comes from the design team. I will translate it.